### DISCUSSION TITLE
Automated Stitching of 360° CGI Images

### DISCUSSION THREAD

**Discussion by miracle524** [community/community#144595] [Programming Help]
Hello everyone,

I'm working on a software development project aimed at creating a solution for stitching together a series of CGI images into a single 360-degree output. I'm looking for contributors who have experience in image processing and are interested in collaborating on this project.

Project Overview:

Objective: Develop reliable software to stitch approximately 56 CGI images per 360-degree view.
Image Specifications: The images have fixed angles and lens settings due to an automated output script, eliminating the need for image analysis or alignment adjustments.
Usage: The software is intended for internal use and should be capable of running on standard office computers.
Technical Details:

Programming Languages: Open to various languages and frameworks (e.g., Python, C++, etc.). The choice can be based on the best fit for performance and ease of use.
Skills Required:
Experience with image processing techniques.
Proficiency in programming languages suitable for this task.
Understanding of CGI and 360-degree imaging concepts.
Strong problem-solving abilities.
Areas Where Help is Needed:

Algorithm Development: Creating or adapting an algorithm to stitch fixed-angle images efficiently.
Software Implementation: Developing a user-friendly application that can be operated without specialized hardware.
Optimization: Ensuring that the software runs reliably and efficiently on office computers.
Testing: Assisting in testing the software with different image sets to ensure robustness.
How to Get Involved:

If you're interested in contributing to this project:

Comment Below or Open an Issue: Share your thoughts, ask questions, or express your interest in collaborating.
Fork the Repository: Once available, you can fork the project repository and start experimenting with the code.
Propose Ideas: I'm open to suggestions on the best approaches, libraries, or tools that could be used.
Share Examples of Your Work: If you have relevant experience, feel free to share examples to help guide the project's direction.
Next Steps:

I'll be setting up a repository to host the project's code and documentation.
Detailed project specifications and requirements will be added to the repository's README file.
A project board will be created to track tasks, issues, and progress.
Goal:

Collaborate to build a robust solution that benefits everyone involved and potentially the wider community interested in CGI image processing.

Looking forward to working with you all!

Best regards,
Jack

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @miracle524!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Reply by miracle524** [reply]
Great, thanks.
